sycl/include/sycl/backend.hpp

```diff
@@ -25,8 +25,12 @@
 #endif
 #endif
 #if SYCL_EXT_ONEAPI_BACKEND_HIP
+#ifdef SYCL_EXT_ONEAPI_BACKEND_HIP_EXPERIMENTAL
+#include <sycl/ext/oneapi/experimental/backend/backend_traits_hip.hpp>
+#else
```

Comment:
This part shouldn't be needed, we have it for CUDA because there's currently two separate interop interface that we support, but there's only one for HIP at the moment. Just `sycl/detail/backend_traits_hip.hpp` should be fine.